.github/workflows/examples-openiotsdk.yaml

```diff
@@ -35,7 +35,10 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-openiotsdk:0.5.101
+            image: atmobica/chip-build-openiotsdk:latest
```

Comment:
We will have to remove this before upstream.

Reply:
Sure